lib/usd/hdMaya/adapters/adapter.cpp

```diff
@@ -13,15 +13,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#include "adapter.h"
-#include "adapterDebugCodes.h"
+#include <hdMaya/adapters/adapter.h>
 
 #include <pxr/base/tf/type.h>
 
 #include <maya/MNodeMessage.h>
```

Comment:
According to the style guidelines, Autodesk + Maya comes before pxr + usd headers.  So these two would be flipped.

Of course, I think we're eventually planning on automating this via clang-format, so if you want to leave this be until we formalize our clang-format file, that's fine - we just need to make sure absolute/relative + <>/"" is handled (since clang-format won't change that.)

There's a few other places where maya vs pxr order is flipped, so I won't bother commenting on all of them.